runners/flink/flink_runner.gradle

```diff
@@ -169,6 +170,17 @@ dependencies {
   validatesRunner project(path: ":runners:core-java", configuration: "testRuntime")
   validatesRunner project(project.path)
   miniCluster "org.apache.flink:flink-runtime-web_2.11:$flink_version"
+  compile project(path: ":model:fn-execution", configuration: "shadow")
+  compile project(path: ":model:pipeline", configuration: "shadow")
+  compile project(path: ":model:job-management:", configuration: "shadow")
+  compile project(":sdks:java:fn-execution")
+  compile project(path: ":vendor:sdks-java-extensions-protobuf", configuration: "shadow")
+  permitUnusedDeclared library.java.jackson_annotations
+  compile library.java.jackson_databind
```

Comment:
Why do we need to add `jackson_databind`?

Reply:
because flink 1.8-flink 1.12 uses jackson_databind. Therefore, instead of adding this dependencies repeatedly  to all the six, i added it here.

Reply:
Oh, I see it now. 👍